packages/upward-js/lib/ContextPath.js

```diff
@@ -2,7 +2,7 @@ const debug = require('debug')('upward-js:ContextPath');
 // Tests for dot-separated strings of "word characters" or slashes.
 // Slashes so that MIME type constants, like "text/html", can be legal values.
 // "Word characters" are [a-zA-Z0-9_].
-const illegalPathChars = /(^[\.\[\]])|[^\.\w\$\/]/;
+const illegalPathChars = /(^[\.\[\]])|[^-\.\w\$\/]/;
```

Comment:
Thank you for doing this, @fooman ! While you're in here, can you please correct another oversight of mine? The `[` and `]` characters should be allowed, in case someone needs to do a context lookup of a list.

```suggestion
const illegalPathChars = /(^[\.\[\]])|[^-\.\w\$\[\]\/]/;
```